### PR TITLE
Added Quarkus GUI React Patternfly Post

### PR DESCRIPTION
**Pull request by nexus-Six**
Created a blog post on how to add a React and Patternfly GUI to a Quarkus app, communicate via REST endpoint and enable dev mode for both runtimes.

** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

File: _posts/2021-03-18-gui-react-patternfly.adoc [hunk 1]
```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: Adding Eyecandy to your Quarkus App with React and Patternfly 
+date: 2021-03-18
```
Comment (maxandersen) [inline _posts/2021-03-18-gui-react-patternfly.adoc:4]: is that your intended publish date?

Comment (nexus-Six) [reply _posts/2021-03-18-gui-react-patternfly.adoc:4]: It was just a random date in the future. Feel free to move it to whenever it fits.

**Comment by Ladicek** [comment]
I'm no expert on frontend development, but `src/main/webapp` is a conventional directory for WAR deployments, so using it might be confusing. Is there any particular reason why you chose it? What about, I don't know, `src/main/frontend`?

**Comment by Ladicek** [comment]
Also, I remembered we have this: https://quarkus.io/blog/quarkus-and-web-ui-development-mode/ They certainly don't have to be the same, but I can see where `webapp` comes from (note that Kabir didn't use `src/main/webapp`, just `webapp`, which is interesting).

And I guess we might have more -- not exactly sure, as I really didn't pay much attention to frontend development in the last ... X years really :-)

**Comment by nexus-Six** [comment]
I choose 'webapp' as it is a common name for the webparts of a java app.  I am absolutely fine with renaming it to 'frontend' to avoid confusion/overlap.

**Comment by nexus-Six** [comment]
Team,

As I am not too familiar with the blog post PR process, please let me know if there is action required on my part? I am fine with modification requests. Or are you currently waiting for the right publishing slot?

**Comment by maxandersen** [comment]
sorry @nexus-Six this fell through the cracks - can you update dates to target monday ? then I'll merge it and it will automatically pop up on monday.

**Comment by nexus-Six** [comment]
No worries @maxandersen . I have just adjusted the publish date and pushed the change.

**Comment by maxandersen** [comment]
okey this is now pushed up with a date for 15th (tomorrow)

sorry for delay - thanks for the blog entry!

**Pull request closed without merging**